mmv1/products/cloudrunv2/Service.yaml

```
@@ -160,6 +160,18 @@ examples:
     ignore_read_extra:
       - 'deletion_protection'
 
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloudrunv2_service_mesh'
+    min_version: 'beta'
+    external_providers: ['time']
+    primary_resource_id: 'default'
+    primary_resource_name: "fmt.Sprintf(\"tf-test-cloudrun-service-%s\", context[\"random_suffix\"])"
+    vars:
+      cloud_run_service_name: 'cloudrun-service'
+      mesh_name: 'mesh'
```

```suggestion
      mesh_name: 'network-services-mesh'
```

The magic module will add `tf-test-` to the mesh_name with `-`. Resources with the prefix `tf-test-` will be recognized by the sweeper and removed.